network-api/networkapi/templates/fragments/libraries/author_card.html

```diff
@@ -8,7 +8,7 @@
             {% if author_profile.image %}
                 {% image author_profile.image fill-182x182 class=profile_image_classes alt=author_profile.name %}
             {% else %}
-                <img src="{% static '_images/author-placeholder.jpg' %}" alt="{{ author_profile.name }}" class="{{ profile_image_classes }}">
+                <img src="{% static '_images/author-placeholder.jpg' %}" width="100" height="100" alt="{{ author_profile.name }}" class="{{ profile_image_classes }}">
```

Comment:
Incidental change -> pipeline linters were complaining that this image tag didn't have `height` and `width` attributes set.